### PR TITLE
Configure Renovate, Update the package from the upper image

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ FROM ubuntu:18.04 as builder
 LABEL maintainer="info@camptocamp.com"
 
 RUN apt-get update && \
+    apt-get update --yes && \
```
Comment (yjacolin) [inline Dockerfile:5]: this is almost the same line as above, can't this be merged?

Comment (sbrunner) [reply Dockerfile:5]: You say it, it's almost the same line, not the same ...

Comment (yjacolin) [reply Dockerfile:5]: :+1:

Comment (arnaud-morvan) [reply Dockerfile:5]: Ok but I  still do not understand why we run `apt update` without `--yes` option.

Comment (sbrunner) [reply Dockerfile:5]: update: update the package dist with version, ...
upgrade: upgrade the installed package to the latest version

Comment (arnaud-morvan) [reply Dockerfile:5]: Ok, so the second one should be `upgrade` instead of `update` as I understand.

Comment (sbrunner) [reply Dockerfile]: Done :-)

**Pull request merged**